constraints.txt

```diff
@@ -267,6 +265,7 @@ SecretStorage==3.3.3
 semsimian==0.2.19
 semsql==0.3.3
 Send2Trash==1.8.3
+setuptools==75.3.0
```

Comment:
> but then again, why would we care about setup tools at runtime?

We don’t have to care about what will happen at runtime, since we will never get to this stage. Setuptools >= 72 prevents some of the packages from being installed at all.